generators/app/index.js

```diff
@@ -231,12 +237,28 @@ module.exports = generators.Base.extend({
       this.destinationPath('.babelrc')
     );
 
-    ['gulpfile.js', 'client', 'config', 'server', 'test'].forEach((f) => {
+    ['gulpfile.js', 'config', 'server', 'test'].forEach((f) => {
       this.fs.copy(
         this.templatePath(f),
         this.destinationPath(f)
       );
     });
+
```

Comment:
sure. although `!this.props.pwa && '**/client/sw-register.js'` does not work. 
`TypeError: pattern.slice is not a function`
so i am ignoring both the files for now and then include them if its a pwa